Quantum-Control-Applications/Superconducting/Single Fixed Transmon/T1.py

```diff
@@ -51,52 +52,66 @@
 #####################################
 #  Open Communication with the QOP  #
 #####################################
-qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+# qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+qmm = QuantumMachinesManager()
```

Comment:
Why did you remove the host?

Reply:
This is my bad, to run the program in my laptop I needed to put the empty parenthesis only